src/Ledger/LedgerForm/LedgerForm.js

```diff
@@ -34,34 +34,20 @@ import {
 const CREATE_LEDGER_TITLE = <FormattedMessage id="ui-finance.ledger.form.title.create" />;
 const EDIT_LEDGER_TITLE = <FormattedMessage id="ui-finance.ledger.form.title.edit" />;
 
-const getPaneFooter = (handleSubmit, pristine, submitting, onCancel) => {
```

Comment:
when you have working code, it would be nice to return footer as it's common style

Reply:
Maybe it makes sense to use a common component? https://github.com/folio-org/stripes-acq-components/blob/master/lib/FormFooter/FormFooter.js